package.json

```diff
@@ -3,15 +3,35 @@
   "private": true,
   "scripts": {
     "link": "lerna link",
-    "postinstall": "lerna bootstrap --no-ci && npm run build && npm run install-hooks",
+    "postinstall": "lerna bootstrap --no-ci && npm run build && npm run install-hooks && cd packages/mobile/ios && pod install",
     "build": "lerna run build --scope=@audius/stems --scope=@audius/common",
     "install-hooks": "mookme init --only-hook --skip-types-selection",
     "lint": "lerna run lint",
-    "lint:fix": "lerna run lint:fix"
+    "lint:fix": "lerna run lint:fix",
+    "WEB========================================": "",
+    "web:dev": "cd packages/web && npm run start:dev",
+    "web:stage": "cd packages/web && npm run start:stage",
+    "web:prod": "cd packages/web && npm run start:prod",
+    "DESKTOP====================================": "",
+    "desktop:dev": "cd packages/web; concurrently -k 'BROWSER=none npm run start:dev' 'wait-on http://localhost:3000 && npm run electron:localhost -- 3000'",
+    "desktop:stage": "cd packages/web; concurrently -k 'BROWSER=none npm run start:stage' 'wait-on http://localhost:3001 && npm run electron:localhost -- 3001'",
+    "desktop:prod": "cd packages/web; concurrently -k 'BROWSER=none npm run start:prod' 'wait-on http://localhost:3002 && npm run electron:localhost -- 3002'",
+    "MOBILE=====================================": "",
+    "ios:dev": "cd packages/mobile && npm run ios:dev",
+    "ios:stage": "cd packages/mobile && npm run ios:stage",
+    "ios:prod": "cd packages/mobile && npm run ios:prod",
```

Comment:
I think we need the `ios:device` commands here too

Reply:
will ensure that this works:
```npm run ios:stage -- --device```

Reply:
Minor adjustment, but this works. I'll update the readme